### DISCUSSION TITLE
Serializing the Rust `Value` type as JSON

### DISCUSSION THREAD

**Discussion by kaathewisegit** [nushell/nushell#13756] [Q&A] [answered]
Is there a way to serialize a `Value` type to JSON in a Rust plugin?  The `serialize` implementation includes types and spans, so `serde_json` with `{name: world}` returns this:

```
{"name":{"String":{"val":"world","span":{"start":106792,"end":106797}}}}
```

Is there a way to get the `{"name": "world"} version somehow?

**Comment by fdncred** [comment, marked as answer]
Can't you do the same thing that `to json` does? I think that means convert nu values to nu-json values and use the nu-json crate?

**Reply by kaathewisegit** [reply]
Ah, I didn't think about looking into the implementation.  Thanks!
